8.3/alpine3.18/fpm-nginx/entrypoints/20-nginx-envsubst-on-templates.sh

```
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+set -e
+
+ME=$(basename $0)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)